src/Validator/Constraints/P0wnedPassword.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the RollerworksPasswordStrengthValidator package.
+ *
+ * (c) Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Rollerworks\Component\PasswordStrength\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+class P0wnedPassword extends Constraint
+{
+    public $message = 'This password was found in a database of compromised passwords. It has been used {{ used }} times. For security purposes you must use something else.';
```

Comment:
This message should be added in translation files, otherwise it won't ever get translation.

Reply:
@gnat42 Can you open a pull request for this?